ru.json

```
@@ -0,0 +1,181 @@
+{
+	"footer": {
+		"description": "Modrinth — программное обеспечение с открытым кодом.",
+		"copyright": "© Guavy LLC",
+		"legal": {
+			"heading": "Legal",
+			"terms": "Условия использования",
+			"privacy": "Политика конфиденацильности",
+			"rules": "Правила"
+		},
+		"resources": {
+			"heading": "Ресурсы",
+			"blog": "Блог",
+			"docs": "Документация"
+		},
+		"buttons": {
+			"privacy": "Настройки конфиденциальности",
+			"theme": {
+				"label": "Изменить тему",
+				"system": "Системная тема",
```

```suggestion
				"system": "Тема системы",
```
или браузера (например, в Firefox можно менять)